fastn-core/src/doc.rs

```diff
@@ -58,6 +58,7 @@ pub async fn interpret_helper(
     loop {
         match s {
             ftd::interpreter::Interpreter::Done { document: doc } => {
+                tracing::info!("done");
```

Comment:
Can we remove these?

Reply:
It's useful in debugging. Tells you which state the interpreter is currently in. This is all hidden behing `fastn --trace ...` anyway.

If you still want these removed, I'll remove them in the next commit (in another PR).